elasticdl/python/worker/worker.py

```diff
@@ -206,28 +232,28 @@ def report_gradient(self, grads):
             else:
                 req.gradient[v.name].CopyFrom(ndarray_to_tensor(g.numpy()))
 
-        # deal with gradients of ElasticDL embedding layer
+        # Accumulate gradients of ElasticDL embedding layer
         # should keep the same order as self.get_trainable_items()
         if self._embedding_layers:
-            grads_edlembedding = grads[origin_var_n:]
+            edl_embedding_grads = grads[origin_var_n:]
 
             bet_number = 0
             for layer in self._embedding_layers:
                 bet_number += len(layer.bet_ids_pair)
```

Comment:
```python
bet_number = sum(map(len, iter(self._embedding_layers)))
```

It seems a little harder to understand.

Reply:
What do you mean by "hard**er**"? I didn't change these lines.

Reply:
Sorry. I mean the code I write seems a little harder to understand.